### DISCUSSION TITLE
Do you mind tagging the new versions of 0.19, 0.20, 0.21, 0.22 in github as well?

### DISCUSSION THREAD

**Discussion by simkimsia** [django-components/django-components#162] [General]
Hi Emil and maintainers team, 

Do you mind tagging in the github repo the various releases of 0.19 all the way to 0.22?

I want to study the differences between each releases so as to better understand how the changes work?

Thank you.

Currently i can only see 0.18 as the latest tag in github repo even tho pypi says it's up to 0.22

**Comment by EmilStenstrom** [comment]
Hi. Thanks for bringing this up. The release process is meant to always add a tag when something is released, but I'd simply forgot about it. I've gone through old commit logs and added correct tags now.
